onnxruntime/core/providers/cuda/nn/batch_norm.cc

```diff
@@ -149,7 +155,7 @@ Status BatchNorm<T, NHWC>::ComputeInternal(OpKernelContext* p_op_kernel_context)
         bn_tensor_desc,
         scale_data,
         b_data,
-        momentum_,
+        1.0 - momentum_,
```

Comment:
Good catch.

I found a a blog for the difference between momentum and exponentialAverageFactor: https://kaixih.github.io/batch-norm/